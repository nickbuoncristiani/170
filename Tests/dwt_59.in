59
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59
35 37 59 6 24 55 58 54 39 5 17 46 28 4 9 25 57 36 11 16 30 52 45 38 53
13
x 1.18227 1.38717 x 1.57723 1.64252 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.18227 x 1.52659 1.55866 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.38717 1.52659 x 1.78321 x 1.63928 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x 1.55866 1.78321 x x 1.41721 x x x x x x x x 1.72389 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.57723 x x x x x 1.93791 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.64252 x 1.63928 1.41721 x x 1.94124 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x 1.93791 1.94124 x x x x x x x x x 1.5406 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.77571 x 1.4373 1.36044 1.21252 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.77571 x 1.86999 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.86999 x x x 1.08767 x x x 1.89052 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.4373 x x x 1.38979 x 1.31746 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.36044 x x 1.38979 x 1.42676 1.95872 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.21252 x 1.08767 x 1.42676 x 1.31866 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.31746 1.95872 1.31866 x x x x 1.34402 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.72389 x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.36892 1.04689 1.31836 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.5406 x x x x x x x x x x x 1.4109 1.36808 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x 1.89052 x x x x x x x x x x x x x x x 1.00012 x x x 1.33898 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x 1.34402 x x x x x x x x x x x x x x x x x 1.94272 x x x x 1.70088 x 1.65814 x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.4109 x x x x 1.18781 1.33015 1.748 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.36808 x x x x x x 1.98776 x x x x x x x x x 1.88889 x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.18781 x x 1.26823 x 1.62231 1.5321 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.33015 x 1.26823 x 1.20453 x 1.53383 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x 1.748 1.98776 x 1.20453 x x 1.88791 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x 1.62231 x x x x 1.57612 1.50955 1.57257 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x 1.5321 1.53383 1.88791 x x x x 1.21675 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.00012 x x x x x x 1.57612 x x 1.34983 x 1.01414 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x 1.50955 x 1.34983 x 1.42465 1.29983 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x 1.57257 1.21675 x 1.42465 x 1.54838 x 1.64254 x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.01414 1.29983 1.54838 x 1.65695 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x 1.33898 x x x x x x x x x x x 1.65695 x x 1.75947 x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.64254 x x x x x x x x x x x x x x 1.80475 1.39901 1.18507 1.07602 x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.94272 x x x x x x x x x x x 1.75947 x x x x x x 1.73378 1.83789 x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.36892 x x x x 1.88889 x x x x x x x x x x x x x 1.62127 x 1.221 x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.04689 x x x x x x x x x x x x x x x x x 1.62127 x 1.07512 x x x x 1.25242 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.31836 x x x x x x x x x x x x x x x x x x 1.07512 x x x x x 1.07154 x x x x x x x 1.40202 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.221 x x x x x x 1.37924 x 1.25569 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.70088 x x x x x x x x x x x x x 1.73378 x x x x x x 1.24486 x 1.68147 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83789 x x x x x x x x 1.86777 x 1.33831 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x 1.65814 x x x x x x x x x x x x x x x x x x 1.24486 x x x 1.88581 x x x x x x x x x 1.50197 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.25242 1.07154 1.37924 x x x x x x x x x x x x 1.13211 x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.68147 1.86777 1.88581 x x x x x x x x x x 1.96507 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.25569 x x x x x x x x x 1.48427 x x x x x x x 1.33227 1.1778 x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.80475 x x x x x x 1.33831 x x x x x x 1.8526 x 1.73019 x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.39901 x x x x x x x x x x x x x 1.92895 x 1.59771 x x x x x x x x 1.54512 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.18507 x x x x x x x x x x x 1.8526 1.92895 x x 1.36448 x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.07602 x x x x x x x x x x 1.48427 x x x x x x x x x x x x 1.46366 1.08432 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.73019 1.59771 1.36448 x x x x 1.0929 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.40202 x x x x x x x x x x x x x 1.97844 x x 1.2619 x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.13211 x x x x x x x 1.97844 x x x x 1.35283 1.99663 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.96507 x x x x x 1.0929 x x x 1.20133 x x x x x x 1.89856 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.50197 x x x x x x x x x x 1.20133 x x x x x x x x 1.0035
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.2619 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.35283 x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.33227 x x x x x x 1.99663 x x x x x 1.0375 1.23391 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.1778 x x x 1.46366 x x x x x x x 1.0375 x 1.57788 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.54512 x 1.08432 x x x x x x x 1.23391 1.57788 x 1.46996 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.46996 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.89856 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.0035 x x x x x x x x
