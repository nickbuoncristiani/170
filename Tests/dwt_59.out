13 10 17 30 32 37 39 37 32 30 17 10 9 8 13
6
17 17
30 30
37 37
39 39 24 25 59 28
9 9
8 11
